### DISCUSSION TITLE
Release Notes?

### DISCUSSION THREAD

**Discussion by benjaminplong** [ERGO-Code/HiGHS#1717] [Q&A]
Are there release notes for different versions? I'm considering upgrading from 1.6 to 1.7, but I would like to understand the changes between the two versions so I can estimate how much work will need done after upgrading.

**Comment by jajhall** [comment]
We're not good at doing this, and one feature of v1.7.0 is that it has the following text

https://github.com/ERGO-Code/HiGHS/releases

**Reply by benjaminplong** [reply]
thank you
